### ISSUE TITLE
more toolbar elements in the Customize Toolbar

### ISSUE THREAD

**Issue by ralf3u** [geany/geany#2690]
It would be so perfect to have those elements in the Customize Toolbar, so one could place them in the toolbar for a fast access:
- Show Line Endings
- Show White Space
- Line Wrapping
- Show Line numbers
- Hide/show menu bar (or even more perfect: menu-symbol, like in Firefox) 

Or, is there a tutorial how to add elements in the Customize Toolbar?

By the way, the White Spaces are too small and nearly invisible.

**Comment by elextr** [comment]
> Or, is there a tutorial how to add elements in the Customize Toolbar?

Afraid its the code.

> By the way, the White Spaces are too small and nearly invisible.

The whitespace dots are drawn by Scintilla which has an option for changing the size but Geany does not use it.  Somebody could add an option to set it.

**Comment by eexpress** [comment]
I want a new butoon for ctrl-shift-d "find current selection in document".

**Comment by ralf3u** [comment]
Nowadays I use Geany like this: There are the tabs on top of the window and nothing else. So, I use Geany without window-bar, without menu-bar, without status bar, without line numbers, without message window, without long line marker, and without toolbar. Because nowadays the toolbar is disabled in my Geany, I don't need any more some symbols in the toolbar. And that is the reason why I close this issue that I opened nearly three years ago. In the reported issue I asked for `Hide/show menu bar`. Last year I found a solution for this. I made a step-by-step-description how to hide/show the menu bar: https://github.com/geany/geany/issues/633#issuecomment-1030634638.
